docs/user_guide/transformation/ArcsinTransformer.rst

```diff
@@ -0,0 +1,79 @@
+.. _reciprocal:
+
+.. currentmodule:: feature_engine.transformation
+
+ArcsinTransformer
+=====================
```

Comment:
the underlining must not be shorter or longer than the title. Otherwise sphinx doesnt like it ;)